### ISSUE TITLE
xgboost TrainingJobAnalytics returns only a single timestamp of rmse

### ISSUE THREAD

**Issue by adarshchavakula** [aws/amazon-sagemaker-examples#945]
My training job completed successfully and I can see the train and validation rmse metrics in the CloudWatch logs. However when I run TrainingJobAnalytics to plot the loss curves, I get the records for only a single timestamp back, not all of them.

```
metric_name = ['train:rmse', 'validation:rmse']
metrics_dataframe = TrainingJobAnalytics(
    training_job_name=job_name,
    metric_names=metric_name
).dataframe() 
```

My output for metric_dataframe looks like this:

	     timestamp          metric_name          value
      0	0.0	        train:rmse           0.309702
      1	0.0	        validation:rmse      0.339424

How can see the complete logs with the all the timestamps?

**Comment by Baijiaoo** [comment]
Hi Adarshchavakula, any solution or ideas about this issue?

**Comment by hsl89** [comment]
Dear SageMaker Community,
                 In our attempts to provide this repository with a better level of support                 going forward, we’re closing issues that were opened prior to the v2 release of the SDK.                 This is because we believe that over time many of the issues                 posted were solved with the latest release or other recent changes to the repo.                 This will help us reallocate resources towards issues that are                more likely to still be relevant today.                Some of the issues experienced now can be resolved by referencing the                v2 guide: https://sagemaker.readthedocs.io/en/stable/v2.html                 In this guide, you can find simple solutions to common notebook errors,                 like the renaming of parameters and classes.                If you believe your issue is still ongoing and you have updated error                messaging or other info, please re-open it, and we investigate the issue.
                Best Regards,
                AWS SageMaker Team

**Comment by yoandinkov** [comment]
Debugged a bit similar scenario, wrote my notes in SO. https://stackoverflow.com/a/69785558/1626392
